web/beacon-app/src/features/projects/routes/ProjectDetailPage.tsx

```diff
@@ -36,13 +36,11 @@ const ProjectDetailPage = () => {
     }
   }, [param, navigate, projectID]);
 
-  // redirect user to project page if project is not found
-
   useEffect(() => {
-    if (!hasProject) {
+    if (!hasTenant || !hasProject) {
```

Comment:
I noticed that when a project detail page is refreshed, the user is re-directed to the projects page. I know that if they switch organizations they should be directed to the projects page of the other organization, but it seems that this is happening even if a user refreshes a project detail page without changing orgs.

Here's a link to a video where I recorded this happening. https://www.awesomescreenshot.com/video/17491916?key=6fe8a6b970112b5d55811e447de2906a

In the video, I also noticed that if a user clicks `Save` in the `Change Owner` modal without selecting a new owner that they get the following error: 
```
new_owner must be a `string` type, but the final value was: `{ "label": "undefined", "value": "undefined" }`.
```